spec/index.bs

```diff
@@ -2227,10 +2229,11 @@ WorkerNavigator includes NavigatorGPU;
 interface GPU {
     Promise<GPUAdapter?> requestAdapter(optional GPURequestAdapterOptions options = {});
     GPUTextureFormat getPreferredCanvasFormat();
+    readonly attribute FrozenArray<DOMString> shadingSoftwareExtensions;
```

Comment:
Editors discussed today and decided we preferred the `readonly setlike` over the `FrozenArray` because it can be queried without searching the array. This line would become `[SameObject] readonly attribute GPUShadingSoftwareExtensions shadingSoftwareExtensions;`.

We also discussed the "interface of booleans that are always true" option as well since it more clearly implies staticness of the supported extension set, but the "always true" bit was weird, and setlike won out for providing an API that less forcibly implies the existence of all wgsl extensions, and matching the GPUSupportedFeatures array.

(There was some thought about the fact that this makes the type the same as GPUSupportedFeatures, but I think we want a separate type for this.)